Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Task.java

```diff
@@ -139,6 +142,17 @@
                     @JoinColumn(name = "userGroup_id", foreignKey = @ForeignKey(name = "FK_task_x_user_userGroup_id")) })
     private List<UserGroup> userGroups;
 
+    /**
+     * This field contains information about folders whose contents are to be
+     * generated in this task.
+     */
+    @ManyToMany(cascade = CascadeType.PERSIST)
+    @JoinTable(name = "generateContents_task_x_folder",
+        joinColumns = @JoinColumn(name = "task_id", foreignKey = @ForeignKey(name = "FK_generateContents_task_x_folder_task_id")),
+        inverseJoinColumns = @JoinColumn(name = "folder_id", foreignKey = @ForeignKey(name = "FK_task_x_folder_folder_id"))
+    )
+    private List<Folder> generateContents;
```

Comment:
I'm sorry, but i don't get the name "generateContents"
It sounds like a function, but is is a list. Is it already generated content? or content for future generation?

Reply:
The javadoc abouve reads: _This field contains information about folders whose contents are to be generated in this task._ This means, when the step is finished, the list of folders will be taken and their contents will be generated. The list contains the folders for whose the on/off switch (see intranet page Generieren von Bildern → 3. Generator automatisch aktivieren) is set to „on“.

Reply:
So it's something like "contentToBeGenerated"?

Reply:
Yes. But I don’t think the active form (“the task generates the following contents”) is less meaningful than the passive form (“the task has the following contents to be generated”). I am not a native speaker of English, but I was taught that English prefers the active form, unlike we Germans tend to use too much passive.

Reply:
I have marked it in review. @matthias-ronge You have told me that you have changed it to 'contentGenerator'. Why this name is again back here?

Reply:
It was named `typeGenerate` before, and I renamed it to `generateContents`. I had not seen your suggestion that day because I didn’t reload the browser page, so my “changed” addressed the fact that I changed it. I first saw your suggestion in this morning. By the way, I find `generateContents` better understandable than `contentGenerator`, because the task will generate the contents when it is completed. The variable holds a list of folders whose contents are to generate, it does *not* contain a list containing generators. So question is if we stay with the name or not. I would like that we have a **final** decision on this because renaming it through three branches is a lot of additional work, so I don’t want to do it several times again. @Kathrin-Huber what is your opinion on this?

Reply:
It's not about active and passive, it's about the fact, that it is a LIST. A list can not DO anything. it IS someting. And i want the name to tell me, WHAT is in the list. If i get it right its a list of folders, from where content is taken. So it should be "contentFolders" because, thats what they are. Don't you think?

Reply:
I find 'contentFolders' name suitable here.